app/services/import_actuals.rb

```diff
@@ -88,8 +89,12 @@ def create_actual
       attrs = @converter.to_h
       assign_default_values(attrs)
 
-      creator = CreateActual.new(activity: @activity, report: @report)
-      result = creator.call(attributes: attrs)
+      creator = if @converter.actual_class == Refund
+        CreateRefund.new(activity: @activity, report: @report, user: @uploader)
+      else
+        CreateActual.new(activity: @activity, report: @report, user: @uploader)
+      end
+      result = creator.call(attributes: attrs.except(:refund_value))
```

Comment:
I'm not following this very well: why are we discarding `:refund_value` at this point?

Reply:
Value is somewhat overloaded, as both the symbol of the Actual Value from the spreadsheet and the value of a field in eithe an Actual or Refund object. Giving these different names is a good idea.

Reply:
Changed a great many of the 'value' variable names to be clearer, explicitly making `:value` (the value of either the Refund or Actual) either `:actual_value` or `:refund_value` when referring to the initial spreadsheet, and only `:value` when it is both those concepts successfully combined (and the only difference is whether it's going into an Actual or a Refund). This is in a separate commit.